en/components/carousel.md

```diff
@@ -12,6 +12,10 @@ Use the Carousel Component to let the user navigate through a collection of imag
 
 <img class="responsive-img" src="../images/carousel_demo.png" srcset="../images/carousel_demo@2x.png 2x" />
 
+## Indicators and Slides
+
+The Carousel lets you define up to 10 slides, where for each one there is a matching indicator dot, allowing the user to navigate to that respective slide.
+
```

Comment:
I think we need to update all images for this topic except for the dos and donts

Reply:
And also add an image after this section that is new.